helm/cloud-provider-vsphere/values.schema.json

```diff
@@ -37,6 +37,28 @@
                 }
             },
             "type": "object"
+        },
+        "kube-vip": {
+            "properties": {
+                "enabled": {
+                    "type": "boolean"
+                }
+            },
+            "type": "object"
+        },
+        "kube-vip-cloud-provider": {
+            "properties": {
+                "cidrGlobal":{
+                    "type": "string"
+                },
+                "enabled": {
+                    "type": "boolean"
+                },
+                "rangeGlobal":{
+                    "type": "string"
+                }
+            },
+            "type": "object"
```

Comment:
I guess it would be good to add defaults

Reply:
Added.